doc/Documentation_Basics.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Documentation_Basics"
+---
+
+##Installation
+- Java SE Development Kit 17.x.x oder höher benötigt
+- in IDE einbinden
+- libGDX wird verwendet, ist aber schon eingebunden und muss nicht extra installiert werden
+
+#### kurzer Einschub: was ist ein ECS?
+- siehe [readme.md](https://github.com/Programmiermethoden/Dungeon/blob/master/doc/ecs/readme.md)
+
+##Struktur des Repos
+
+Relevante Klassen zum Einstieg:
+Package "starter"
+Game.java (Haupt-Steuerung des Dungeon)
+- beinhaltet main
+- verwaltet die anderen Controller und Systeme
+- alle Systeme befinden sich im „package ecs→ systems“
+	VelocitySystem -> aktualisiert Position von Entitäten 
+	DrawSystem -> zum zeichnen von Objekten
+	PlayerSystem -> zur Steurung der Figur
+	AISystem  -> AI steurung
+	CollisionSystem  -> Kollisions Prüfung auf zwischen zwei Entitäten
+	HealthSystem -> Schaden, den Entität erleidet
+
+wichtige Methoden:
+setup(): 
+- Aufruf beim Anwendungsstart
+- initialisirung und Konfigurierung Objekte, die vor dem Spielstart existieren müssen (z.B Systeme,  Controller, Spielfigur)
+- levelAPI Erstellung und laden der neuen Level
+
+frame() 
+- Wird zu Beginn eines jeden Frame aufgerufen
+
+render()
+- Game-Loop des Spiels. Zeichnet den Dungeon neu und ruft die eigene Implementierung auf.
+
+onLevelLoad()
+- Aufruf, wenn das Laden eines neuen Levels erfolgt
+
+Class DesktopLauncher:
+- erstellt das Hauptfenster des Spiels mit bestimmten Maßen (maxWidth" und "maxHeight Beeinflussen die Größe) inklusive Logo  und Spieltitel
+
+Class LibgedxSetup:
+ create()
+- mithilfe der SpriteBatch werden alle Sprites (individuelle Zeichnungen) in einzigen Zeichenaufruf gebündelt
+
+--------------------------------------------------------------
+Package "level"
+class LevelAPI 
+- Level Erstellung und Verwaltung
+- lädt neues Level mit loadLevel()
+- update() zeichnet neues Level
+
+--------------------------------------------------------------
+Package "graphic"
```

Comment:
Auch hier: Erklären was eine ANimation ist, wie ich sie anlege und wie die funktioniert.
Den rest auslagern. In der Basic Variante brauch ich garnit wissen wie man das mit dem nextAnimationPath funktioniert.